cowprotocol/cowswap
# Test NPS

# Summary

This PR does three things:
-  Move all the logic of appZi to a library, so the component logic is simpler, and we make it easier to add new appzi feedback elements in a central place
- Added a function which opens the NPS feedback form
- 

# Not part of this PR
- Styles for the NPS form
- Logic on how it should be opened

# To Test
1. Trade, wait until the trade is executed
2. Observe how the NPS form is opened
